vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -2439,28 +2452,6 @@ def process_chunk(chunk_start, chunk_end, skip_result_store=False):
             staged_hidden_states.copy_(hidden_states, non_blocking=True)
             staged_router_logits.copy_(router_logits, non_blocking=True)
 
-            # If there are shared experts but we are not using a modular kernel,
```

Comment:
note: it was actually impossible to reach this case before. We currently disable `shared_expert_stream` for the `chunked_impl`